### PR TITLE
fixes #9525 - add array equality assertion methods

### PR DESCRIPTION
**Pull request by stbenjam**
Fixes the random case that occurred in http://ci.theforeman.org/job/test_katello_core/11499/database=postgresql,ruby=1.9.3/testReport/junit/%28root%29/Katello__SystemTest/test_available_and_applicable_errta/

### CODE REVIEW

**Comment by stbenjam** [comment]
Jenkins: http://ci.theforeman.org/view/Katello/job/test_katello_pull_request/5789/

File: test/katello_test_helper.rb [hunk 1]
```diff
@@ -171,6 +171,18 @@ def mock_active_records(*records)
       record.stubs(:reload).returns(record)
     end
   end
+
+  def equal_arrays(array1, array2)
+    array1.sort == array2.sort
+  end
+
+  def assert_equal_arrays(array1, array2)
+    assert equal_arrays(array1, array2)
```
Comment (ehelms) [inline test/katello_test_helper.rb]: What about just doing:

```
assert_equal array1.sort, array2.sort
```

And refute equal for the other function without needing the `equal_arrays` function?

Comment (stbenjam) [inline test/katello_test_helper.rb]: Makes no difference to me, I'll change it

**Comment by stbenjam** [comment]
@ehelms Updated

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
ACK

**Pull request merged**